crates/sui-protocol-config/src/lib.rs

```diff
@@ -488,4 +488,12 @@ impl ProtocolConfig {
             _ => panic!("unsupported version {:?}", version),
         }
     }
+
+    // Add more arguments for different test scenarios
+    pub fn get_for_move_publish(max_function_definitions: Option<usize>) -> Self {
```

Comment:
@mystenmark this is where `ProtocolConfig` gets a bit ugly but not sure how to do it otherwise.
This change is very simple but as we go on testing limits, being able to manipulate the config is vital and more functions could be added.
Unless you have a better model...

Reply:
Maybe some sort of prefix/suffix `test` for this?

Reply:
done